src/components/TestPage/TestPage.jsx

```diff
@@ -40,28 +44,47 @@ export const TestPage = () => {
     }
   };
 
+  const startTimer = () => {
+    if (time.m === 1) {
+      tick();
+      toggle();
+    }
+  };
+
   return (
     <Scaffold minutes={time.m} seconds={time.s}>
+      <input onKeyDown={startTimer}></input>
       <div className="test">
-        <button
-          aria-label="play-button"
-          disabled={isActive}
-          onClick={toggle}
-          type="button"
-        >
-          Play
-        </button>
-        <button
-          aria-label="pause-button"
-          disabled={!isActive}
-          onClick={toggle}
-          type="button"
-        >
-          Pause
-        </button>
-        <button aria-label="restart-button" onClick={restart} type="button">
-          Restart
-        </button>
+        {isPauseShowing && (
+          <Button
+            onClick={toggle}
+            aria-label="pause-button"
+            disabled={!isActive}
+            variant="primary"
+            iconName="pause"
+            text="Pause"
+          />
+        )}
+        {!isPauseShowing && (
+          <Button
+            onClick={toggle}
+            aria-label="play-button"
+            disabled={isActive}
+            variant="primary"
+            iconName="start"
+            text="Start"
+          />
+        )}
+        {!isPauseShowing && (
+          <Button
+            onClick={restart}
+            aria-label="restart-button"
+            disabled={isActive}
+            variant="secondary"
+            iconName="restart"
+            text="Restart"
+          />
+        )}
```

Comment:
If you want to save yourself a few lines you can do
```jsx
        {isPauseShowing ? (
          <Button { ...props } /> // Pause button
        ) : (
          <>
            <Button { ...props } /> // Play button
            <Button { ...props } /> // Restart button
          </>
        )}
```

Reply:
Makes sense.  
```js
<>
   <Button { ...props } /> // Play button
   <Button { ...props } /> // Restart button
 </>
```

For this part ^ are we wrapping it with an empty tag or should it be a div/fragment maybe?

Reply:
Probably a div, since we do need the block with those two buttons to be centered

Reply:
I gotcha. We already have a div that can center the the whole block of all the buttons, but another div to wrap the two buttons could be useful for generally manipulating the buttons inside the div.